talk/src/contexture/components.js

```diff
@@ -5,7 +5,14 @@ import {observer, inject} from 'mobx-react'
 import {Flex} from '../flex'
 import './components.css'
 
-export let Facet = inject('tree')(observer(({tree, node, hide = {}, ...props}) => (
+export let InjectTreeNode = inject(
```

Comment:
can't this be done like `{tree: t1}, {tree = t1, path}` ?

Reply:
Seems like it can be done:
<img width="1100" alt="screen shot 2018-02-22 at 6 01 39 pm" src="https://user-images.githubusercontent.com/417016/36569107-769d6dc8-17fa-11e8-9f4c-eccc882b986e.png">

Reply:
done :)